crates/bevy_pbr/src/render/mesh_view_types.wgsl

```diff
@@ -19,6 +19,7 @@ struct ClusterableObject {
 
 const POINT_LIGHT_FLAGS_SHADOWS_ENABLED_BIT: u32   = 1u;
 const POINT_LIGHT_FLAGS_SPOT_LIGHT_Y_NEGATIVE: u32 = 2u;
+const POINT_LIGHT_FLAGS_VOLUMETRIC_BIT: u32        = 4u;
```

Comment:
Genuine question: why is this 4 rather than 3?

Reply:
`1 << 2` is `0b100` aka "the third flag" but this bit pattern is the literal `4u`. wgsl doesn't support defining consts via bitwise ops which is why you will sometime see flags like `const FLAG = 268435456` (2^28) or some other large number for high numbered flags.

Reply:
1 << 2 is 0b100 aka "the third flag" but this bit pattern is the literal 4u. wgsl doesn't support defining consts via bitwise ops which is why you will sometime see flags like const FLAG = 268435456 (2^28) or some other large number for high numbered flags.